scripts/test_runner.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+ENV_PRODUCT="${ENV_PRODUCT}"
+ENV_TFVARS="${ENV_TFVARS}"
+TESTDIR="${TESTDIR}"
+TESTTAG="${TESTTAG}"
+CMD="${CMD}"
+EXPECTEDVALUE="${EXPECTEDVALUE}"
+VALUEUPGRADED="${VALUEUPGRADED}"
+CHANNEL="${CHANNEL}"
+INSTALLVERSIONORCOMMIT="${INSTALLVERSIONORCOMMIT}"
+SUCUPGRADEVERSION="${SUCUPGRADEVERSION}"
+TESTCASE="${TESTCASE}"
+WORKLOADNAME="${WORKLOADNAME}"
+DESCRIPTION="${DESCRIPTION}"
+DEPLOYWORKLOAD="${DEPLOYWORKLOAD}"
+SONOBUOYVERSION="${SONOBUOYVERSION}"
+
+if [ -z "${TESTDIR}" ]; then
+    printf "\n\nTESTDIR is not set\n\n"
+    exit 1
+fi
+
+if [ -z "${IMGNAME}" ]; then
+   printf "\n\nIMGNAME is not set\n\n"
+   exit 1
+fi
+
+
+case "$TESTDIR" in
+     upgradecluster|versionbump|mixedoscluster|dualstack|createcluster)
+      printf "\n\nRunning tests for %s\n\n" "${TESTDIR} on ${ENV_PRODUCT}"
+        ;;
+    *)
+        printf "\n\nTESTDIR is not a go test package\n\n"
```

Comment:
Here should print the value of the TESTDIR variable instead of just the string "TESTDIR" itself

Reply:
Tk you. Yeah

Reply:
Actually no 

Since I was reassign, it is correct

Reply:
But If I will remove the reasgin I will refactor everything

Reply:
Wait shouldn't it just be: `printf "\n\n%s is not a go test package\n\n" "${TESTDIR}"` kind of like how it is above?

Reply:
yes. sorry, forget what i said . was to tired lol